app/styles/app.scss

```diff
@@ -20,51 +22,72 @@ $light-grey-banner: #eeeeee;
 $dark-banner: #607D8B;
 
 
+@mixin floating-panel {
+  background: #ffffff;
+  padding: 20px;
+  box-shadow: 0 5px 5px -5px rgba(0,0,0,0.3);
+  margin: 15px 15px;
+}
+
+
 /* Global */
 
-h2:hover > .headerlink {
-    opacity: 1;
+h2:hover >.headerlink {
+  opacity: 1;
+}
+
+
+.info {
+  @include floating-panel;
+  background-color:#daf5ff;
 }
 
-section {
+.case-detail {
+  @include floating;
+  @include panel;
+  background: #ffffff;
+  margin: 15px 15px;
+  padding: 20px;
+
+  /* Global */
+
+  h2:hover > .headerlink {
+    opacity: 1;
+  }
+
+  section {
```

Comment:
These should be global, not confined to section-detail.

Reply:
+1